step.sh

```diff
@@ -283,6 +284,7 @@ fi
 echo_details "* device_farm_project: $device_farm_project"
 echo_details "* test_package_name: $test_package_name"
 echo_details "* test_type: $test_type"
+echo_details "* billing_method: $billing_method"
```

Comment:
also around line 306 please add the following code to validate the expected option was passed:
```
options=("METERED"  "UNMETERED")
validate_required_input_with_options "billing_method" "${billing_method}" "${options[@]}"
```